receiver/httpdreceiver/testdata/httpd.conf

```diff
@@ -0,0 +1,555 @@
+#
```

Comment:
Consider removing unused/commented lines from this file, to help future readers understand what is actually relevant.

Reply:
I could be wrong, but I think it's pretty standard for application config files to have a lot of settings commented out so that they can easily be enabled if necessary.